packages/assertion-monitor/README.md

```diff
@@ -0,0 +1,46 @@
+# Assertion Monitor
+
+This tool is designed to check for chains that have not posted an assertion in the last seven days. Read more about assertions [here](https://docs.arbitrum.io/how-arbitrum-works/assertion-tree).
```

Comment:
I think this section covers the relevant information better: https://docs.arbitrum.io/how-arbitrum-works/inside-arbitrum-nitro#arbitrum-rollup-protocol